docs/policy_templates_data/policy_templates.json

```diff
@@ -931,6 +931,34 @@
         ]
       }
     },
+    "AWSSecretsManagerGetSecretValueAccess": {
+      "Description": "Grants permissions to GetSecretValue for the specified AWS Secrets Manager secret",
+      "Parameters": {
+        "SecretArn": {
+          "Description": "The ARN of the secret to grant access to"
+        }
+      },
+      "Definition": {
+        "Statement": [
+          {
+            "Effect": "Allow",
+            "Action": [
+              "secretsmanager:GetSecretValue"
+            ],
+            "Resource": {
+              "Fn::Sub": [
+                "${secretArn}",
```

Comment:
Our policy templates follow the pattern of forcing scoping the resource to the current account and region for tighter security. Will following this pattern work for your use case? For example, you could change the SecretArn parameter to SecretName. Then do this:

```
"Resource": {
  "Fn::Sub": [
    "arn:${AWS::Partition}:secretsmanager:${AWS::Region}:${AWS::AccountId}:secret:${secretName}",
    {
      "secretName": {
        "Ref": "SecretName"
      }
    }
  ]
}
```

Reply:
Discussed use case offline. There is a use case for needing to specify the full ARN.